ansible/templates/usb/boot/menu.lst.j2

```diff
@@ -6,7 +6,6 @@ terminal composite
 variable os_console vga
 splashimage=/boot/splash.xpm.gz
 
-{% if usb_type == "cn" %}
 title Compute Node 64-bit
```

Comment:
Are we sure we don't want to distinguish between first node (full) USB key install and the other USB keys? At least by some comment.